WordPress/WordPressShareExtension/ShareExtensionEditorViewController.swift

```diff
@@ -981,9 +981,11 @@ extension ShareExtensionEditorViewController {
         alertController.popoverPresentationController?.sourceView = richTextView
         alertController.popoverPresentationController?.sourceRect = CGRect(origin: position, size: CGSize(width: 1, height: 1))
         alertController.popoverPresentationController?.permittedArrowDirections = .any
-        present(alertController, animated: true, completion: { () in
-            UIMenuController.shared.hideMenu()
-        })
+        present(alertController, animated: true) {
+            for interaction in self.richTextView.interactions {
```

Comment:
This code hides the menu that occasionally shows up when tapping on media, which still works with the new code:

https://github.com/user-attachments/assets/c47281d2-4c8f-414d-acee-ddc41638bc8b